lib/tasks/yaml_db_anonymizer.rake

```diff
@@ -1,11 +1,23 @@
+require 'zlib'
+
 namespace :db do
   namespace :data do
-    desc 'Dump anonymized database contents to db/data.yml'
-    task :dump_anonymized => :environment do
+    def anonymize_helper
       format_class = ENV['class'] || "YamlDb::Anonymizer::Helper"
       require format_class.split('::').first.underscore
       helper = format_class.constantize
-      SerializationHelper::Base.new(helper).dump File.join(Rails.root, 'db', "data.#{helper.extension}")
+    end
```

Comment:
Good catch. Since it isn't very complex, and the functionality is only needed for the rake tasks, I'd just put it into the tasks.